### ISSUE TITLE
Exception: input_max_range must be larger than input_min_range

### ISSUE THREAD

**Issue by amolgupta** [googlecodelabs/tensorflow-style-transfer-android#4]
Tried running the codelab and getting the following exception:

`05-24 06:48:50.906 31372-31397/org.tensorflow.demo E/TensorFlowInferenceInterface: Failed to run TensorFlow inference with inputs:[input, style_num], outputs:[transformer/expand/conv3/conv/Sigmoid]
05-24 06:48:50.907 31372-31397/org.tensorflow.demo E/AndroidRuntime: FATAL EXCEPTION: inference
                                                                     Process: org.tensorflow.demo, PID: 31372
                                                                     java.lang.IllegalArgumentException: input_max_range must be larger than input_min_range.
                                                                     	 [[Node: transformer/contract/conv1/Relu_eightbit_quantize_transformer/contract/conv1/InstanceNorm/batchnorm/add_1 = QuantizeV2[T=DT_QUINT8, mode="MIN_FIRST", _device="/job:localhost/replica:0/task:0/cpu:0"](transformer/contract/conv1/InstanceNorm/batchnorm/add_1, transformer/contract/conv1/Relu_eightbit_min_transformer/contract/conv1/InstanceNorm/batchnorm/add_1, transformer/contract/conv1/Relu_eightbit_max_transformer/contract/conv1/InstanceNorm/batchnorm/add_1)]]
                                                                         at org.tensorflow.Session.run(Native Method)
                                                                         at org.tensorflow.Session.access$100(Session.java:48)
                                                                         at org.tensorflow.Session$Runner.runHelper(Session.java:285)
                                                                         at org.tensorflow.Session$Runner.run(Session.java:235)
                                                                         at org.tensorflow.contrib.android.TensorFlowInferenceInterface.run(TensorFlowInferenceInterface.java:142)
                                                                         at org.tensorflow.demo.StylizeActivity.stylizeImage(StylizeActivity.java:597)
                                                                         at org.tensorflow.demo.StylizeActivity.access$1400(StylizeActivity.java:67)
                                                                         at org.tensorflow.demo.StylizeActivity$3.run(StylizeActivity.java:551)
                                                                         at android.os.Handler.handleCallback(Handler.java:751)
                                                                         at android.os.Handler.dispatchMessage(Handler.java:95)
                                                                         at android.os.Looper.loop(Looper.java:154)
                                                                         at android.os.HandlerThread.run(HandlerThread.java:61)`

**Comment by markmcd** [comment]
Can you point to the code you executed and some info about your environment (e.g. Android Studio version, OS, device type & OS etc)

**Comment by amolgupta** [comment]
I used the StylizeActivity from codelab-finish branch.

Environment:
OSX v10.11.6 
Android Studio: v2.3.2
Device: Nexus 5X running Android 7.1.2
gradle version 3.3
buildToolsVersion "25.0.3"


Another error log i see just before the exception is:
`05-24 06:54:19.993 3132-3132/? E/art: No implementation found for long org.tensorflow.contrib.android.RunStats.allocate() (tried Java_org_tensorflow_contrib_android_RunStats_allocate and Java_org_tensorflow_contrib_android_RunStats_allocate__)
`

**Comment by markmcd** [comment]
I've not been able to reproduce this using AS 3.0 canary, AS 2.3.2, AS 2.3.1 with a Pixel or a 5X, though I'm sure I saw this happen at the I/O codelab booth on a Pixel too.

If you want to get unblocked, you can find some workaround instructions on [this issue](https://github.com/tensorflow/tensorflow/issues/8550) (this code is forked from the TF repo).

I think we should move discussion over to that thread too, so I'll close this issue.

**Comment by amolgupta** [comment]
Thanks @markmcd . I compiled using AS 3.0 canary and it worked fine.

**Comment by snownus** [comment]
@markmcd when I run Faster RCNN, I came across error:

2017-12-06 11:05:49.646516: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1120] Creating TensorFlow device (/device:GPU:0) -> (device: 0, name: Tesla K40m, pci bus id: 0000:09:00.0, compute capability: 3.5)
2017-12-06 11:06:30.110730: W tensorflow/core/framework/op_kernel.cc:1192] Invalid argument: input_max_range must be larger than input_min_range.

**Comment by snownus** [comment]
@markmcd  +more error information.

2017-12-06 11:06:30.110756: W tensorflow/core/framework/op_kernel.cc:1192] Invalid argument: input_max_range must be larger than input_min_range.
	 [[Node: SecondStagePostprocessor/BatchMultiClassNonMaxSuppression/MultiClassNonMaxSuppression/ClipToWindow_1/Area/mul_eightbit/SecondStagePostprocessor/BatchMultiClassNonMaxSuppression/MultiClassNonMaxSuppression/ClipToWindow_1/Area/sub_1/quantize = QuantizeV2[T=DT_QUINT8, mode="MIN_FIRST", _device="/job:localhost/replica:0/task:0/device:CPU:0"](SecondStagePostprocessor/BatchMultiClassNonMaxSuppression/MultiClassNonMaxSuppression/ClipToWindow_1/Area/sub_1/_817, SecondStagePostprocessor/BatchMultiClassNonMaxSuppression/MultiClassNonMaxSuppression/ClipToWindow_1/Area/mul_eightbit/SecondStagePostprocessor/BatchMultiClassNonMaxSuppression/MultiClassNonMaxSuppression/ClipToWindow_1/Area/sub_1/min/_819, SecondStagePostprocessor/BatchMultiClassNonMaxSuppression/MultiClassNonMaxSuppression/ClipToWindow_1/Area/mul_eightbit/SecondStagePostprocessor/BatchMultiClassNonMaxSuppression/MultiClassNonMaxSuppression/ClipToWindow_1/Area/sub_1/max/_821)]]
2017-12-06 11:06:30.110760: W tensorflow/core/framework/op_kernel.cc:1192] Invalid argument: input_max_range must be larger than input_min_range.
